lib/robots/dor_repo/ocr/fetch_files.rb

```diff
@@ -34,7 +34,7 @@ def abbyy_input_path
         end
 
         def ocrable_filenames
-          @ocrable_filenames ||= ocr.required? && ocr.possible? ? ocr.filenames_to_ocr : []
+          @ocrable_filenames ||= ocr.filenames_to_ocr
         end
```

Comment:
we don't need to bother doing all of this since the ocrWF will not start if it's not possible now